src/Controller/Api/MajorVersion/RequirementsController.php

```diff
@@ -84,11 +83,11 @@ public function getRequirementsByMajorVersion(string $version, Request $request)
             'json',
             SerializationContext::create()->setGroups(['data'])
         );
-        $response = new JsonResponse($json, 200, [], true);
-        $response->headers->set(AbstractSessionListener::NO_AUTO_CACHE_CONTROL_HEADER, 'true');
-        $response->setEtag(md5($json));
-        $response->isNotModified($request);
-        return $response;
+        $jsonResponse = new JsonResponse($json, 200, [], true);
+        $jsonResponse->headers->set(AbstractSessionListener::NO_AUTO_CACHE_CONTROL_HEADER, 'true');
+        $jsonResponse->setEtag(md5($json));
+        $jsonResponse->isNotModified($request);
+        return $jsonResponse;
```

Comment:
I'm not sure the rename here adds much in context.  I'm not against it, really, I just don't see that it adds value.

Reply:
Yep, `$response ` was already fine, not sure if I can prevent this change. Ofc I can disable that rule but it also did good changes in other places.